### PR TITLE
Notebooks cleanup: Remove old out of proc markdown option

### PR DESCRIPTION
**Pull request by chlafreniere**
We've defaulted to using the in-proc markdown for the past 4 months, and have not gotten any bugs that prevent us from not being able to move to the in-proc markdown option by default.

Also should improve code coverage slightly 😄

### CODE REVIEW

File: build/lib/stats.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ function submitAllStats(productJson, commit) {
                 }
             */
             appInsights.defaultClient.trackEvent({
-                name: 'monacoworkbench/packagemetrics',
+                name: `${productJson.quality !== 'stable' ? 'adsworkbench' : 'monacoworkbench'}/packagemetrics`,
```
Comment (kevcunnane) [inline build/lib/stats.js]: This looks unrelated. Did you intend to have this in this PR? Can we ignore / remove?

Comment (kevcunnane) [reply build/lib/stats.js]: Maybe a merge from master will remove this?

Comment (chlafreniere) [reply build/lib/stats.js]: Hmm this just did a merge. Happens everytime I build.

Comment (chlafreniere) [reply build/lib/stats.js]: Reverted
